pySDC/helpers/problem_helper.py

```diff
@@ -112,7 +127,7 @@ def get_finite_difference_matrix(
             if steps[i] < 0:
                 A_1d += coeff[i] * sp.eye(size, k=size + steps[i])
     else:
-        raise NotImplementedError(f'Boundary conditions {bc} not implemented.')
+        raise NotImplementedError(f'Boundary conditions \"{bc}\" not implemented.')
```

Comment:
You don't have to put the escape characters here, since you use `'` to define the string. This is enough :

```python
raise NotImplementedError(f'Boundary conditions "{bc}" not implemented.')
```